site/en/blog/devtools-tips-12/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: >
+  DevTools Tips: How to inspect and modify CSS animations
+description: >
+  Use the Animations tab to inspect and modify CSS animations, transitions, and more.
+layout: 'layouts/blog-post.njk'
+date: 2022-08-11
+authors:
+  - sofiayem
+hero: 'image/NJdAV9UgKuN8AhoaPBquL7giZQo1/ZcQHNwVHsQ3dIVNWbUAH.png'
+alt: >
+  DevTools Tips hero logo
+tags:
+  - css
+  - devtools
+  - devtools-tips
+---
+
+The Chrome DevTools **Animations** tab is a powerful tool that helps you slow down, replay, nail CSS animation timings, and see the results right away.
```

Comment:
I'm wondering about how "nail CSS animation timings" might be understood. It reads a bit strangely to me.